lib/Froxlor/Cron/Http/Apache.php

```diff
@@ -907,7 +907,8 @@ protected function getVhostContent($domain, $ssl_vhost = false)
 			if (filter_var($domain['ip'], FILTER_VALIDATE_IP, FILTER_FLAG_IPV6)) {
 				$ipport = '[' . $domain['ip'] . ']:' . $domain['port'] . ' ';
 			} else {
-				$ipport = $domain['ip'] . ':' . $domain['port'] . ' ';
+				//$ipport = $domain['ip'] . ':' . $domain['port'] . ' ';
+				$ipport = '*:' . $domain['port'] . ' ';
```

Comment:
Same here...just no

Reply:
Yep, see the other wrong change comment, i'm checking on how to edit the pull request